File: app/src/main/java/com/digicoffer/lauditor/Meetings/MonthCalander/CalendarItem.java
package com.digicoffer.lauditor.Meetings.MonthCalander;

import java.time.LocalDate;

public class CalendarItem {
    public final LocalDate date;
    public final String label;

    public CalendarItem(LocalDate date) {
        this.date = date;
        this.label = null;
    }

    public CalendarItem(String label) {
        this.label = label;
        this.date = null;
    }

    public boolean isHeader() {
        return label != null;
    }
}

File: app/src/main/java/com/digicoffer/lauditor/Meetings/MonthCalander/CalendarItem.kt
package com.digicoffer.lauditor.Meetings.MonthCalander

import java.time.LocalDate

class CalendarItem {
    @JvmField
    val date: LocalDate?
    @JvmField
    val label: String?

    constructor(date: LocalDate?) {
        this.date = date
        this.label = null
    }

    constructor(label: String) {
        this.label = label
        this.date = null
    }

    fun isHeader(): Boolean {
        return label != null
    }
}
